server.go

```diff
@@ -167,6 +167,9 @@ type Server struct {
 	//   * ErrBrokenChunks
 	ErrorHandler func(ctx *RequestCtx, err error)
 
+	// Trace is invoked for various events generated by this server.
+	Trace ServerTrace
```

Comment:
I think this field should have more docs; and obviously we can't say that a struct can be invoked

Reply:
I felt like most of the documentation would be redundant with what I included in the documentation for `ServerTrace` itself. I'm happy to expand here though if you have concrete suggestions.